src/presentational/Dropdown.jsx

```diff
@@ -17,9 +21,10 @@ export default class Dropdown extends Component {
   render() {
     const options = this.props.options.map(option =>
         <option
-          key={option.value}
-          value={option.label}>
-            {option.label}
+          // the disjunctions allow for the case where options are classrooms (right-hand side)
```

Comment:
Sorry it's a bit confusing I guess. I originally written this component with the idea that the `options` were stored in separate file with `value/label` properties, i.e. `constants/utils.js`. However I wanted to be able to reuse the dropdown for classrooms as well, which don't have a `value/label` structure, hence the _ad hoc_ disjunction.